ektelo/query_matrix.py

```diff
@@ -0,0 +1,123 @@
+from ektelo.matrix import EkteloMatrix, Identity, Ones, VStack, Kronecker
```

Comment:
Do I remember correctly that this file was going to be removed? It is only used in one place as far as I can tell:
```
ektelo/client/selection.py:15:from ektelo import matrix, query_matrix
ektelo/client/selection.py:276:        return query_matrix.Total(self.domain_shape[0])
```

Reply:
Many of the things in this file were moved to workload.py, the remaining things are not really used throughout the codebase yet, but I think they will be in the future.  They will be needed when I start implementing the HDMM optimization operators and such.  Also the Hierarchical class can eventually replace the current version and it will offer an efficiency boost by doing things in a matrix-free fashion

Reply:
I will delete this for this branch and bring it back in later